UPGRADE.md

```diff
@@ -942,9 +942,9 @@ Now you can start your project as usual with `./start`.
 
 ## Updating jQuery
 
-The jQuery version used by IHP has changed. We're switching from `3.2.1` to `3.6.0`. jQuery `3.2.1` has some known security vulnerabilities, so it's recommended that you follow these update steps.
```

Comment:
This refers to an older version of IHP. Ideally you could add a new entry for the next IHP release and add a note on the jquery upgrade

Reply:
Thanks, sorry, this line confused me, will fix it 🙂